### DISCUSSION TITLE
qiankun 新 logo 发布啦！

### DISCUSSION THREAD

**Discussion by kuitos** [umijs/qiankun#1987] [Announcements]
一年前，我们发布了 qiankun 的[首版 logo](https://github.com/umijs/qiankun/discussions/1304)。

但它似乎并未达到预期的品牌传播效果，一些 qiankun 的用户也表达了 logo 辨识度不够的问题：
> 只看logo，很难想到这是qiankun 😂

基于此，我们在[ qiankun 3.0 ](https://github.com/umijs/qiankun/discussions/1378)启动之际，同时也启动了新的 logo 的设计议程，并于近日敲定了新 logo 的第一版方案：

<p align="center">
  <image src="https://gw.alipayobjects.com/mdn/rms_84469a/afts/img/A*DNvvRYQnb70AAAAAAAAAAAAAARQnAQ" width="30%">
</p>

关于新 logo 的设计灵感，听听 logo 的第一作者 @chessl 的分享：

> 形体上继承了前 Logo 的基本思想，以扁平的方型和圆形为基础元素组成了乾卦 ☰、坤卦 ☷，但对这两个字的定义进行了重新思考，以「坤」为圆而「乾」为方。
>
> 私以为「坤」是将各类应用包裹的容器，才是「地」；而「乾」是将整个容器里所有应用链接到更大宇宙网络的通道，才是「天」。同时继续使用了同心圆来组成「坤」卦，保留了任意技术栈的子应用环环相扣却互不影响的寓意，也使「乾坤」更符合世界主流文化理解中的球体宇宙的形象和体现其容器的功能。
>
>「乾」卦代表的通道从容器内部突出连向更大宇宙，和圆形的「坤」组合在一块并将其低置也更像「qiankun」的首字母「Q」。在这个基础上将所有的线条形状进行了等宽处理，只使用了黑白灰，简化了视觉元素更容易被记忆，加强了视觉感受，更具现代感和科技感。

如果你对新的 logo 有任何的意见或建议，欢迎在下方留言互动，我们期待能与社区一起参与 qiankun 的每一次变化🧑‍💻

**Comment by Deturium** [comment]
是你吗，蚊香蝌蚪？

![image](https://user-images.githubusercontent.com/19731097/156304995-a56ad829-c833-4b86-bedd-bbea00048bdf.png)

**Reply by ice-zjchen** [reply]
是我鸭，是我鸭
![image](https://user-images.githubusercontent.com/2605406/156305316-a48d68be-6a06-49de-b605-0cf8b814c816.png)

**Comment by kuitos** [comment]
新配色版本
<image width="30%" src="https://user-images.githubusercontent.com/5206843/156369489-cf708a6a-1937-4dd9-895b-c6d17156c493.png">

**Reply by hotWater-me** [reply]
新logo 感觉看一眼还是忘🙄

**Comment by csgajcr** [comment]
个人感觉第一眼看 LOGO 有点感受不到主体，有点抽象，但配色和设计灵感感觉很不错 👍 👍 。

**Comment by yongchaozeng** [comment]
个人感觉乾卦 ☰、坤卦 ☷这种大部分人都不太明白，还不如用乾坤中的天地日月这些含义，更能让人明白。不过这个黑白logo挺好看的，就是没什么联系。也挺好的

**Comment by renwofei423** [comment]
大部分程序员更关心性能问题，嘿嘿

**Comment by cXiaof** [comment]
我觉得还不如旧版更"具现代感和科技感"

**Comment by SnuggleE** [comment]
具体什么时候发布呢？

**Comment by kuitos** [comment]
final version
![qiankun3-logo](https://github.com/user-attachments/assets/cfe2f034-4df5-40cd-ba30-fe4187afacd5)
